### ISSUE TITLE
Bar Charts

### ISSUE THREAD

**Issue by nataliekatsaros** [nataliekatsaros/QMB210-15-Group3#6]

**Comment by jackdawid** [comment]
How many juniors rated the food 7

**Comment by Taylorpenzo** [comment]
?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nataliekatsaros** [comment]
he commented on the github u have to comment on your ticket and have a back and forth for credit

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Taylorpenzo** [comment]
What do you mean?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by AlexBunker** [comment]
I did the gender of respondents and whether or not they live on campus is that okay?

**Comment by Taylorpenzo** [comment]
Their are 5 seniors, 18 juniors, 6 sophomores and 14 freshman in this survey analysis. 

All 14 freshman use  15 meal swipes 

All 6 sophomores use 7 swipes

Out of the 18 juniors; 1 using 7 swipes, 1 using 15 swipes and 16 without a meal plan 

Out of the 5 seniors: 1 using 7 swipes and 4 having no meal plan

**Comment by nataliekatsaros** [comment]
> I did the gender of respondents and whether or not they live on campus is that okay?

Yes, it would be cool if you did whether they lived on campus and whether they thought the food was healthy too. It might give more information about the topic.

**Comment by nataliekatsaros** [comment]
> How many juniors rated the food 7

Check the new survey data, then input these numbers into your graphs.

**Comment by nataliekatsaros** [comment]
> Their are 5 seniors, 18 juniors, 6 sophomores and 14 freshman in this survey analysis.
> 
> All 14 freshman use 15 meal swipes
> 
> All 6 sophomores use 7 swipes
> 
> Out of the 18 juniors; 1 using 7 swipes, 1 using 15 swipes and 16 without a meal plan
> 
> Out of the 5 seniors: 1 using 7 swipes and 4 having no meal plan

Good analysis. Use this information in your video if you used it. The data varying by grade is a good variable that you can use grades instead of gender in your comparisons.

**Comment by AlexBunker** [comment]
Could I piggyback off of Taylor's graphs by adding some pie charts maybe? Other than that my ideas were what she did. Maybe I could do something with healthier options vs non healthy options? Feels like I'm trying to find a needle in a haystack rn

**Comment by nataliekatsaros** [comment]
> Could I piggyback off of Taylor's graphs by adding some pie charts maybe? Other than that my ideas were what she did. Maybe I could do something with healthier options vs non healthy options? Feels like I'm trying to find a needle in a haystack rn

If you need help coming up with more ideas I can help you to make graphs with new info. Remember you guys need 4 charts (choose 4 out of these: bar chart,, vertical chart, ogive, continuous, pareto).
